src/main/java/request/HttpRequest.java

```diff
@@ -19,7 +28,29 @@ public String method() {
   public String path() {
     return requestLine().split(" ")[1];
   }
+
   public String version() {
     return requestLine().split(" ")[2];
   }
+
+  public String getValueForHeader(String key) {
+    return headers.get(key);
+  }
+
+  public int contentLength() {
+    String length = getValueForHeader("Content-Length");
+    int rtnLength = 0;
+    if (length != null) {
+      rtnLength = Integer.parseInt(length);
```

Comment:
Although we're not going to be running business-critical things on this server, it _might_ be good to start thinking about "how will an external system break my code".